plugins/pulp_rpm/plugins/distributors/yum/publish.py

```diff
@@ -265,8 +275,15 @@ def __init__(self, transfer_repo, publish_conduit, config, distributor_type, **k
 
         last_published = publish_conduit.last_publish()
         last_deleted = repo.last_unit_removed
-        force_full = config.get('force_full', False)
-        date_filter = None
+
+        # if a filter is passed in, assume force_full and use the filter as the
+        # date filter
+        if association_filters:
+            force_full = True
+            date_filter = association_filters
+        else:
+            force_full = config.get('force_full', False)
+            date_filter = None
```

Comment:
I spent some time trying to remove `@skip_broken` from `PublisherTests` to get the tests running, but I am not having much luck.